examples/basic/index.html

```diff
@@ -35,7 +35,7 @@
 </head>
 
 <body>
-
+  <canvas id="webgl"></canvas>
```

Comment:
This is helpful if you want to overlay the canvas with something else, such as a loading screen. And it is more obvious.